src/main/resources/data/computercraft/lua/rom/programs/id.lua

```diff
@@ -13,16 +13,32 @@ if sDrive == nil then
     end
 
 else
+    local bAudio = disk.hasAudio(sDrive)
+    if bAudio then
+        print("Audio medium")
```

Comment:
Maybe instead of doing "Audio medium", we either display `Has audio track "the title"` or `Unknown audio track`?

Dunno - "Audio medium" is correct, but also feels a bit of a clunky descriptor?

Reply:
This was exactly my problem with this program... Hmm `Has audio track "the title"` or `Has untitled audio track` ?